share/commands/remote-run

```diff
@@ -20,17 +20,18 @@
 
 VERBOSITY=""
 COLOR="-C"
+DEBUG_CLASS="-D debug"
 
 while getopts "iIvdc" opt; do
   case $opt in
     i|I)
-      VERBOSITY="-I -D debug"
+      VERBOSITY="-I ${DEBUG_CLASS}"
       ;;
     v)
-      VERBOSITY="-v"
+      VERBOSITY="-v ${DEBUG_CLASS}"
       ;;
     d)
-      VERBOSITY="-d"
+      VERBOSITY="-d ${DEBUG_CLASS}"
       ;;
     c)
       COLOR=""
```

Comment:
Yes it could be cool if you could do it.
Is the output for remote agent the same as for a local agent ?